src/zerocash/Address.cpp

```diff
@@ -11,26 +11,14 @@
  * @copyright  MIT license (see LICENSE file)
  *****************************************************************************/
 
-#include <cryptopp/osrng.h>
-using CryptoPP::AutoSeededRandomPool;
-
-#include <cryptopp/eccrypto.h>
-using CryptoPP::ECP;
-using CryptoPP::ECIES;
-
-#include <cryptopp/oids.h>
-namespace ASN1 = CryptoPP::ASN1;
-
-#include <cryptopp/filters.h>
-using CryptoPP::StringSink;
-using CryptoPP::StringStore;
+#include "zcash/NoteEncryption.hpp"
 
 #include "Zerocash.h"
 #include "Address.h"
 
 namespace libzerocash {
 
-PrivateAddress::PrivateAddress(const std::vector<unsigned char> a_sk, const std::string sk_enc) {
+PrivateAddress::PrivateAddress(const uint256 &a_sk, const uint256 &sk_enc) {
```

Comment:
Which PR? I only see this one at https://github.com/zcash/zcash/pulls that is relevant.